priv/templates/phx.gen.live/form_component.html.leex

```diff
@@ -10,5 +10,7 @@
   <%= input %>
   <%= error %>
 <% end %>
-  <%%= submit "Save", phx_disable_with: "Saving..." %>
+  <div>
```

Comment:
This is not part of this issue but I'll make the generated form a bit more beautiful.

### before
![image](https://user-images.githubusercontent.com/27698968/95569273-01ce4200-0a2e-11eb-9109-231b920a7e90.png)


### after 
![image](https://user-images.githubusercontent.com/27698968/95569221-f1b66280-0a2d-11eb-9b0c-6acc083ea5bf.png)

Just copied from here: https://github.com/phoenixframework/phoenix/blob/c8883af5582a38496e4b7e45e05d3a4d759a6caa/priv/templates/phx.gen.html/form.html.eex#L12-L14